packages/ethicalMetrics/src/isApiUp.ts

```diff
@@ -0,0 +1,13 @@
+import * as url from "url";
+import fetch from "node-fetch";
+export async function isApiUp(): Promise<boolean> {
```

Comment:
I would instead use this function internally in the ethical metrics module on every API request.

You could add a function parameter for the timeout to wait before determining is up or not